docs/source/examples/substrafl/go_further/run_mnist_cyclic.ipynb

```diff
@@ -0,0 +1,1113 @@
+{
```

Comment:
<p>If not, follow the instructions described <a href="https://docs.substra.org/en/stable/substrafl_doc/substrafl_overview.html#installation" target="_blank">here</a>. (missing coma)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>